main/menus.js

```diff
@@ -40,6 +42,24 @@ const cogMenuTemplate = [
   {
     type: 'separator'
   },
+  {
+    label: 'Open Video…',
+    accelerator: 'Cmd+o',
+    click: () => {
+      closeAllCroppers();
+      dialog.showOpenDialog({
+        filters: [
+          {name: 'Videos', extensions: ['mov', 'mp4', 'm4v']},
```

Comment:
We already define the extensions here: https://github.com/wulkano/kap/blob/0cbff512718ed3f31bdbc0913364df57deb8d2fe/main/tray.js#L12 We should put it in a common place to we only have to update it in one place. Maybe a `/main/common/constants.js` file? You'll have to add the dot here instead: https://github.com/wulkano/kap/blob/0cbff512718ed3f31bdbc0913364df57deb8d2fe/main/tray.js#L25